### PR TITLE
:memo: readme

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

**Comment by fabienjuif** [comment]
Visible here: https://github.com/fabienjuif/roll/tree/fabienjuif-patch-1

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
 # Roll
+> Simply choose your dices, and roll them!
+
+This application help us play with multiple dices at once.
```
Comment (delphinemillet) [inline README.md]:
```suggestion
This application helps us play with multiple dices at once.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+## 🎲 Test it!
+Just go to the [website](https://fabienjuif.github.io/roll/).
+
+## 🤔 Why are you developping this app?
```
Comment (delphinemillet) [inline README.md]:
```suggestion
## 🤔 Why are we developping this app?
```

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+This is a nice pretext to play around react hooks and PWA.
+
+## 🤝 Contributors
+Feel free to add yourself in this section!
+ - [Fabien JUIF](https://github.com/fabienjuif) - Dev
+ - [Delphine MILLET](https://github.com/delphinemillet) - Reviews
```
Comment (delphinemillet) [inline README.md]: ...

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+
+
+## 🛠 Hack it!
+Here are the principals commands:
```
Comment (delphinemillet) [inline README.md]:
```suggestion
Here are the main commands:
```

File: README.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+1. Install dependencies: `yarn`
+2. Run de dev server: `yarn start`
```
Comment (delphinemillet) [inline README.md]:
```suggestion
2. Run the dev server: `yarn start`
```

File: README.md [hunk 6]
```diff
@@ -1 +1,27 @@
...
+3. Run the build: `yarn deploy:build` 
+4. Publish on your gh-pages: `yarn gh-pages -d build`
+
+If you want to check your code quality, you can run `yarn ci`, this is the same script that will be run from CircleCI for each commits.
```
Comment (delphinemillet) [inline README.md]:
```suggestion
If you want to check your code quality, you can run `yarn ci`, this is the same script that will be run from CircleCI for each commit.
```

**Review by delphinemillet — CHANGES_REQUESTED** [review]

**Review by delphinemillet — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1 +1,27 @@
...
+## 🎲 Test it!
+Just go to the [website](https://fabienjuif.github.io/roll/).
+
+## 🤔 Why are we developping this app?
+This is a nice pretext to play around react hooks and PWA.
+
+## 🤝 Contributors
+Feel free to add yourself in this section!
+ - [Fabien JUIF](https://github.com/fabienjuif) - Dev
+ - [Delphine MILLET](https://github.com/delphinemillet) - Reviews
```
Comment (fabienjuif) [inline README.md]:
```suggestion
 - [Delphine MILLET](https://github.com/delphinemillet)
```

File: README.md [hunk 8]
```diff
@@ -1 +1,27 @@
...
+
+## 🎲 Test it!
+Just go to the [website](https://fabienjuif.github.io/roll/).
+
+## 🤔 Why are we developping this app?
+This is a nice pretext to play around react hooks and PWA.
+
+## 🤝 Contributors
+Feel free to add yourself in this section!
+ - [Fabien JUIF](https://github.com/fabienjuif) - Dev
```
Comment (fabienjuif) [inline README.md]:
```suggestion
 - [Fabien JUIF](https://github.com/fabienjuif)
```

File: README.md [hunk 9]
```diff
@@ -1 +1,27 @@
...
+Just go to the [website](https://fabienjuif.github.io/roll/).
+
+## 🤔 Why are we developping this app?
+This is a nice pretext to play around react hooks and PWA.
+
+## 🤝 Contributors
+Feel free to add yourself in this section!
+ - [Fabien JUIF](https://github.com/fabienjuif)
+ - [Delphine MILLET](https://github.com/delphinemillet)
+
```
Comment (fabienjuif) [inline README.md:16]:
```suggestion
```

**Pull request merged**